### PR TITLE
Add conversions for DATACONTROL, SAVE, STAT, VARIABLE, and ABILITYCAT…

### PR DESCRIPTION
**Pull request by alexperovich**
…EGORY files

### CODE REVIEW

File: Data/core_rulebook/cr__datacontrols.lua [hunk 1]
```diff
@@ -0,0 +1,7127 @@
+-- Converted From LST file data\pathfinder\paizo\roleplaying_game\core_rulebook\cr__datacontrols.lst
```
Comment (ChadNedzlek) [inline Data/core_rulebook/cr__datacontrols.lua:1]: This whole file appears to be nonsense. :-) I wonder what it's useful for.

File: Data/core_rulebook/cr__saves.lua [hunk 2]
```diff
@@ -0,0 +1,35 @@
+-- Converted From LST file data\pathfinder\paizo\roleplaying_game\core_rulebook\cr__saves.lst
+-- From repository https://github.com/pcgen/pcgen at commit 11ceb52482855f2e5f0f6c108c3dc665b12af237
+DefineSave({
```
Comment (ChadNedzlek) [inline Data/core_rulebook/cr__saves.lua:3]: Heavens! A lst file that appears to contain something actually useful!  Though we could probably just... hard code this. :-p

File: Data/core_rulebook/cr__datacontrols.lua (hunk 1 above)
Comment (alexperovich) [reply Data/core_rulebook/cr__datacontrols.lua:1]: its defining how the `FACT:` tags work. None of it is documented though.

File: Data/core_rulebook/cr__saves.lua (hunk 2 above)
Comment (alexperovich) [reply Data/core_rulebook/cr__saves.lua:3]: I know, I was surprised when I saw a file with actual content in it.

File: Data/core_rulebook/cr_abilities.lua [hunk 3]
```diff
@@ -10411,7 +10411,7 @@ DefineAbility({
   Definitions={
     {
       Name="RangePenaltyValue",
-      InitialValue=0,
+      InitialValue="0",
```
Comment (ChadNedzlek) [inline Data/core_rulebook/cr_abilities.lua:10414]: Why all the quotes all of the sudden?

**Pull request merged**